translations/nl-be.json

```diff
@@ -124,7 +124,7 @@
   "document-add": "Document toevoegen",
   "document-link-old": "Reeds bezorgde documenten koppelen",
   "document-delete": "Document verwijderen",
-  "document-edit": "Bewerk document details",
+  "document-edit": "Wijzigen",
```

Comment:
document-edit is op 2 plaatsen gebruikt.
Zowel de titel van de modal als de naam van de actie
Die titel moest niet gewijzigd worden naar wijzigen ? 
Volgens figma moet die titel "Bewerk document" zijn, ik zou deze key daar op aanpassen
Ik zou dan dan in de actie de key "edit" gebruiken, wat al vertaald naar Wijzigen
"edit-2" = Bewerken indien dat beter is